src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
+/* eslint-disable no-shadow */
 'use strict';
 
-// write code here
+function parseSalary(salary) {
+  return parseFloat(salary.replace(/[^0-9.-]+/g, ''));
+}
+
+function sortList(list, property) {
+  const itemsArray = Array.from(list);
+
+  itemsArray.sort((a, b) => {
+    const valueA = parseSalary(a.getAttribute(`data-${property}`));
+    const valueB = parseSalary(b.getAttribute(`data-${property}`));
+
+    return valueB - valueA;
+  });
+
+  const parent = list[0].parentElement;
```

Comment:
Consider adding a check to ensure that the list is not empty before accessing `list[0].parentElement`. This will prevent potential runtime errors if the list is empty.